dotcom-rendering/src/web/components/ImageComponent.tsx

```diff
@@ -368,6 +368,7 @@ export const ImageComponent = ({
 										displayCredit={element.displayCredit}
 										shouldLimitWidth={shouldLimitWidth}
 										isOverlayed={true}
+										isMainMedia={true}
```

Comment:
Is the prop value of `isMainMedia` always going to be true here? If so, it might be safer to just pass it down. I can imagine a future when things are moved around or copy pasted and this potentially creating hard to find bugs.